design-toolkit/components/src/components/button/styles.ts

```diff
@@ -132,21 +132,19 @@ const BaseButtonStyles = tv({
         'size-large:max-h-xxl size-large:min-h-xxl size-large:min-w-xxl size-large:max-w-xxl',
         'size-medium:max-h-[28px] size-medium:min-h-[28px] size-medium:min-w-[28px] size-medium:max-w-[28px]',
         'size-small:max-h-xl size-small:min-h-xl size-small:min-w-xl size-small:max-w-xl',
-        'size-large:max-h-[20px] size-large:min-h-[20px] size-large:min-w-[20px] size-large:max-w-[20px]',
+        'size-xsmall:max-h-[20px] size-xsmall:min-h-[20px] size-xsmall:min-w-[20px] size-xsmall:max-w-[20px]',
```

Comment:
Why are we defining heights at all? Min widths and max widths are ok, but a big red flag on defining heights (even if they're min/max)